src/index.js

```diff
@@ -137,14 +155,88 @@ const getPointsLeftByDay = async (sprint, start) => {
       pointsLeftByDay[i] = 0;
     }
   }
+  log.info(JSON.stringify({ numDaysSinceSprintStart }));
+
+  if (!INCLUDE_WEEKENDS) {
+    // remove weekend entries
+    let index = 0;
+    for (
+      const cur = moment(start);
+      index < pointsLeftByDay.length;
+      cur.add(1, "days")
+    ) {
+      if (isWeekend(cur)) {
+        pointsLeftByDay.splice(index, 1);
+      } else {
+        index++;
+      }
+    }
+  }
+
   return pointsLeftByDay;
 };
 
-const generateChart = async (data, labels, filenamePrefix) => {
-  const pointsPerDay = data[0] / (labels[labels.length - 1] - 1);
-  const constantLine = labels.map(
-    (label) => data[0] - pointsPerDay * (label - 1)
+/** generate the guideline. A flat line is shown across weekends if {@link INCLUDE_WEEKENDS} is set to true */
+const getGuideline = (start, end, initialPoints, numberOfWeekdays) => {
+  const pointsPerDay = initialPoints / numberOfWeekdays;
+
+  log.info(
+    JSON.stringify({
+      initialPoints,
+      numberOfWeekdays,
+      pointsPerDay,
+    })
+  );
+
+  const guideline = [];
+  const cur = moment(start);
+  const afterEnd = moment(end).add(1, "days"); // to include the end day data point
+  let prevDayIsWeekday = false;
+  for (let index = 0; cur.isBefore(afterEnd); index++, cur.add(1, "days")) {
+    // if not including the weekends, just skip over the weekend days
+    if (!INCLUDE_WEEKENDS) {
+      while (isWeekend(cur)) {
+        cur.add(1, "days");
+      }
+    }
+
+    if (index === 0) {
+      guideline[index] = initialPoints;
```

Comment:
Hmm. Do we really need index in this loop? Or can we just use .push?

Reply:
oh good idea!

Reply:
hmm actually it will result in more duplicated code, cause lines 197-201 and 210 will have to be repeated outside before the loop and also an additional `cur.add(1,"days")` line